tests/load/test_load_case.py

```diff
@@ -23,3 +23,19 @@ def test_load_case_rank_model_version(case_obj, adapter):
 
     assert loaded_case["rank_model_version"] == case_obj["rank_model_version"]
     assert loaded_case["sv_rank_model_version"] == case_obj["sv_rank_model_version"]
+
+
+def test_load_case_limsid(case_obj, adapter):
+
```

Comment:
Could you add a very short doctring for this function? One line would be sufficient